### DISCUSSION TITLE
Welcome to My Scheduled Tasks Discussions!

### DISCUSSION THREAD

**Discussion by Timthreetwelve** [Timthreetwelve/MyScheduledTasks#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
Hi,
My name is vikas kumar . I am a software developer in India . Particularly I work in dot net technology in IT .

I have started to work closely with team now and it's quite fascinating to use github actions in that . I would like to discuss on how friendly is to  work with github actions for windows forms application . 

Also want to know how test case is prepared for windows forms . Its quite difficult in case of integration test case

**Comment by Timthreetwelve** [comment]
I don't have any experience with windows forms application. 

Perhaps there is a better place to ask for advice.
